packages/itwinui-css/src/radio-tile/radio-tile.scss

```diff
@@ -147,8 +146,8 @@
   height: $iui-icons-xl;
   display: block;
   margin: 0 auto;
-  padding-top: round($iui-baseline * 0.5);
-  padding-bottom: $iui-baseline;
+  margin-top: round($iui-baseline * 0.5);
+  margin-bottom: $iui-baseline;
```

Comment:
Because `border-box` does include `padding` into `width` or `height.
Read more here
https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing#values

I say its much better if you leave 2 explicit rules rather than merging those, better readability. Also, `auto` is not the same, as actually the default `margin` value is `0`.
https://developer.mozilla.org/en-US/docs/Web/CSS/margin#formal_definition

Reply:
The line before these defines the margin already though.

`margin: 0 auto;`

I just thought it was weird to define the margin and then redefine `margin-top` and `margin-bottom`.

Reply:
oh, right.. Havent seen that one. Then makes sense, I guess

Reply:
Updated